package.json

```diff
@@ -90,28 +104,26 @@
     "babel-core": "6.26.0",
     "babel-jest": "22.1.0",
     "babel-preset-env": "1.6.1",
-    "bundlesize": "0.15.3",
     "coveralls": "3.0.0",
     "danger": "3.0.5",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
     "flow-bin": "0.63.1",
     "graphql": "0.11.7",
     "graphql-tag": "2.6.1",
+    "husky": "^0.14.3",
     "jest": "22.1.1",
     "jsdom": "11.5.1",
     "lint-staged": "6.0.0",
-    "pre-commit": "1.2.2",
     "preact": "^8.2.7",
     "preact-compat": "^3.17.0",
     "prettier": "1.10.2",
     "react": "16.2.0",
     "react-dom": "16.2.0",
     "react-test-renderer": "16.2.0",
     "recompose": "0.26.0",
+    "recursive-rename": "^2.0.0",
     "rimraf": "2.6.2",
-    "rollup": "0.54.1",
```

Comment:
do we not need this anymore? Does typescript handle all of the bundling?

Reply:
tsc does 100% of the transpilation - no more bundles (discussed with @jaydenseric in the thread)

Reply:
rollup added back for bundlesize only